mkdocs/docs/HPC/only/gent/2023/shinx.md

```diff
@@ -0,0 +1,113 @@
+# New Tier-2 cluster: `shinx`
+
+In October 2023, a new pilot cluster was added to the HPC-UGent Tier-2 infrastructure: `shinx`.
+
+This page provides some important information regarding this cluster, and how it differs from the clusters
+it is replacing (`swalot` and `victini`).
+
+If you have any questions on using `shinx`, you can [contact the {{ hpcteam }}]({{ hpc_support_url }}).
+
+For software installation requests, please use the [request form](https://www.ugent.be/hpc/en/support/software-installation-request).
+
+---
+
+## `shinx`: generic CPU cluster
+
+`shinx` is a new CPU-only cluster.
+
+It replaces `swalot`, which will be retired on **Wednesday 01 November 2023**,
+and `victini`, which will be retired on **Monday 05 February 2024**.
+
+It is primarily for regular CPU compute use.
+
+This cluster consists of 48 workernodes, each with:
+
+* 2x 96-core AMD EPYC 9654 (Genoa @ 2.4 GHz) processor;
+* ~360 GiB of RAM memory;
+* 400GB local disk;
+* NDR-200 InfiniBand interconnect;
+* RHEL9 as operating system;
+
+To start using this cluster from a terminal session, first run:
+```
+module swap cluster/shinx
+```
+
+You can also start (interactive) sessions on `shinx` using the [HPC-UGent web portal](../../../web_portal.md).
+
+### Differences compared to `swalot` and `slaking`.
+
+#### CPUs
+
+The most important difference between `shinx` and `swalot`/`victini` workernodes is in the CPUs:
+while `swalot` and `vicitini` workernodes featured *Intel * CPUs, `shinx` workernodes have `AMD Genoa` CPUs.
+
+Although software that was built on a `swalot` or `vicitini` workernode with compiler options that enable architecture-specific
+optimizations (like GCC's `-march=native`, or Intel compiler's `-xHost`) might still run on
+a `shinx` workernode, it is recommended to recompile the software to benefit from the support for
+`AVX-512` vector instructions (which is missing on `swalot`).
```

Comment:
This might not be very meaningful in the future for people who did not read that swallow was replaced by shinx.